tests/darwin/client_test.py

```diff
@@ -0,0 +1,117 @@
+from pathlib import Path
+from typing import Any, Dict
+
+import pytest
+import responses
+from requests.exceptions import HTTPError
+
+from darwin.client import Client
+from darwin.config import Config
+from darwin.exceptions import InsufficientStorage, Unauthorized
+
+
+def describe_put():
```

Comment:
While not an extensive list of tests (there are several use cases missing) it does cover for the very basic usage of the PUT method. 

Also, first set of tests for client !

Reply:
love them